packages/slicemachine-core/src/filesystem/manifest.ts

```diff
@@ -4,6 +4,7 @@ import { FileContent, SMConfigPath } from "./paths";
 export interface Manifest {
   apiEndpoint: Endpoints.ApiEndpoint;
   storybook?: string;
+  libraries?: string[];
```

Comment:
to go a little further, do you think the manifest file should be more like the AuthConfig ?
Having function such as "createOrDefault"

Reply:
Also, i'm pretty sure I took this model from the UI, so there might be a wrong model on the "env" part of the ui

Reply:
As we didn't update this file after creating it, for now this is less relevant